python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/system.schema.yml

```diff
@@ -36,6 +36,10 @@ keys:
                   type: str
                   convert_types:
                     - int
+                  description: "'vrf' and 'ingress_default' are mutual exclusive."
+                ingress_default:
```

Comment:
I think we should two option under ingress i.e. `default` and `enabled`.
Because we can have vrf with ingress but without default.
```
system control-plane
   ip access-group ingress default acl4_4
   ip access-group acl4_3 in
   ip access-group ingress vrf mahesh in <-----------
   ip access-group acl4_2 vrf red in
   ip access-group acl4_2 vrf red_1 in
   ip access-group acl4_5 vrf red_3 in
   ip access-group ingress vrf red_4 in
```

Reply:
or you can add the description for acl_name to use ingress as acl since it is required key

Reply:
Added description.